app/helpers/katello/products_helper.rb

```diff
@@ -32,28 +32,5 @@ def objectify(record, content_types = nil)
         :repos => repos
       }
     end
-
-    # Retrieve a hash of products that are accessible to the user.
-    # This will be determined from the readable & editable products provided in the options.
-    # @param [Hash] options the options to construct the product hash with
-    # @option options [String] :readable_products List of readable products
-    # @option options [String] :editable_products List of editable products
-    # @option options [String] :content_types (Optional) Filter repos by listed types
-    def get_products(options)
-      options[:content_types] ||= [Repository::TYPES]
-      if @product_hash.nil?
-        @product_hash = {}
-        options[:readable_products].sort_by(&:name).each do |prod|
-          repos = prod.repos(current_organization.library).where(:content_type => options[:content_types])
-            .sort { |a, b| a.name <=> b.name }
-          repos = repos.map { |repo| {:name => repo.name, :id => repo.id} }
-          if repos.any?
-            @product_hash[prod.id] = {:name => prod.name, :repos => repos, :id => prod.id,
-                                      :editable => options[:editable_products].include?(prod)}
-          end
-        end
-      end
-      @product_hash
-    end
   end
 end
```

Comment:
app/helpers/katello/products_helper.rb <- I dont wanna remove the full file as its not connected to this PR. I am just removing an unused method because this PR will break without that,